packages/teams-js/src/private/videoEx.ts

```diff
@@ -246,11 +258,26 @@ export namespace videoEx {
    * Sending error notification to host
    * @beta
    * @param errorMessage - The error message that will be sent to the host
+   * @param errorLevel - The error level that will be sent to the host
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  function notifyError(errorMessage: string, errorLevel: ErrorLevel = ErrorLevel.Warn): void {
+    sendMessageToParent('video.notifyError', [errorMessage, errorLevel]);
+  }
+
+  /**
+   * @hidden
+   * Sending fatal error notification to host. Call this function only when your app meets fatal error and can't continue.
+   * The host will stop the video pipeline and terminate this session, and optionally, show an error message to the user.
+   * @beta
+   * @param errorMessage - The error message that will be sent to the host
    *
    * @internal
    * Limited to Microsoft-internal use
    */
-  function notifyError(errorMessage: string): void {
-    sendMessageToParent('video.notifyError', [errorMessage]);
+  export function notifyFatalError(errorMessage: string): void {
+    notifyError(errorMessage, ErrorLevel.Fatal);
```

Comment:
Your exported functions should be calling ensureInitialized() and checking video.isSupported() (see other functions in this file)

Reply:
added, thanks